sample-code/src/server.ts

```diff
@@ -48,8 +55,31 @@ app.get('/embedding', async (req, res) => {
 });
 
 app.get('/orchestration/:sampleCase', async (req, res) => {
+  const sampleCase = req.params.sampleCase;
+  const func =
+    sampleCase === 'simple'
+      ? orchestrationChatCompletion
+      : sampleCase === 'template'
+        ? orchestrationTemplating
+        : sampleCase === 'inputFiltering'
+          ? orchestrationInputFiltering
+          : sampleCase === 'outpttFiltering'
+            ? orchestrationOutputFiltering
+            : sampleCase === 'requestConfig'
+              ? orchestrationRequestConfig
+              : orchestrationChatCompletion;
```

Comment:
[pp] Consider extracting this to a separate function and either using switch or if statements to avoid the nesting. For better readability.

Reply:
Haha yeah I just got inspired by Tom & Deeksha writing [this code](https://github.com/SAP/ai-sdk-js/blob/main/packages/langchain/src/openai/util.ts#L165) 😉 

Any preference what looks better? Simply switch? Or maybe rather a dict?

Reply:
used a dict now